Dockerfile

```diff
@@ -1,17 +1,18 @@
-FROM python:3.12-rc-alpine
+FROM python:3.12-alpine3.17
```

Comment:
With this change, I am getting 403s during fare checks. However, I reverted it back to `rc-alpine` and have not reached any 403s. Can you confirm this @dmytrokoren?

Reply:
I have been running for 5 days without 403 on the 3.12-alpine3.17

After 2 intervals 403 error goes away... but let's keep 3.12-rc-alpine, I'm okay with it.

Reply:
Interesting... I reached a few 403s right away (enough to make the fare check unsuccessful, but none with `rc-alpine`.